src/app/portlets/dot-porlet-detail/dot-portlet-detail.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
I know this component can have a way better implementation I don't have the time right now, I use as much as possible of @alfredo-dotcms work.